src/script/api/script_group.cpp

```diff
@@ -174,24 +180,45 @@
 	return ::Group::Get(group_id)->statistics.profit_last_year;
 }
 
-/* static */ uint32 ScriptGroup::GetCurrentUsage(GroupID group_id)
+struct EfficiencyRating {
+	OverflowSafeInt32 potential;
+	OverflowSafeInt32 delivered;
+	EfficiencyRating & operator+=(const EfficiencyRating &other)
```

Comment:
extra space, and lack of spaces, as per existing examples
```suggestion
	EfficiencyRating &operator += (const EfficiencyRating &other)
```

maybe wants a newline above as well